build/git_revision.go

```diff
@@ -169,45 +174,62 @@ func (gr *GitRevision) Build(ctx context.Context) (string, error) {
 		installDir = tmpDir
 		gr.pathsToRemove = append(gr.pathsToRemove, installDir)
 	}
+	gr.log().Printf("install dir is %q", installDir)
 
 	// copy license file on best effort basis
-	dstLicensePath := filepath.Join(installDir, "LICENSE.txt")
-	srcLicensePath := filepath.Join(repoDir, "LICENSE.txt")
-	altSrcLicensePath := filepath.Join(repoDir, "LICENSE")
-	if _, err := os.Stat(srcLicensePath); err == nil {
-		err = gr.copyLicenseFile(srcLicensePath, dstLicensePath)
-		if err != nil {
-			return "", err
-		}
-	} else if _, err := os.Stat(altSrcLicensePath); err == nil {
-		err = gr.copyLicenseFile(altSrcLicensePath, dstLicensePath)
-		if err != nil {
-			return "", err
-		}
+	// default to installDir if LicenseDir is not set
+	licenseDir := gr.LicenseDir
+	if licenseDir == "" {
+		licenseDir = installDir
+	}
+	gr.log().Printf("attempting to copy license file to %q", licenseDir)
+	if err := gr.copyLicenseIfExists(repoDir, licenseDir); err != nil {
+		return "", err
 	}
 
 	gr.log().Printf("building %s (timeout: %s)", gr.Product.Name, buildTimeout)
 	defer gr.log().Printf("building of %s finished", gr.Product.Name)
 	return bi.Build.Build(buildCtx, repoDir, installDir, gr.Product.BinaryName())
 }
 
+func (gr *GitRevision) copyLicenseIfExists(repoDir string, dstDir string) error {
+	licenseFiles := []string{"LICENSE.txt", "LICENSE"}
+
+	for _, file := range licenseFiles {
+		srcPath := filepath.Join(repoDir, file)
+		if _, err := os.Stat(srcPath); err == nil {
+			gr.log().Printf("found license file at %q", srcPath)
+			dstPath := filepath.Join(dstDir, file)
```

Comment:
```suggestion
			dstPath := filepath.Join(dstDir, "LICENSE.txt")
```

As per my earlier comment the hope is to standardise on this naming convention.